prov/usnic/src/usdf_fabric.c

```diff
@@ -116,13 +186,19 @@ usdf_validate_hints(struct fi_info *hints, struct usd_device_attrs *dap)
 		    fattrp->prov_version != USDF_PROV_VERSION) {
 			return -FI_ENODATA;
 		}
+
+		devn = get_devname(version, dap);
+
 		if (fattrp->name != NULL &&
-                    strcmp(fattrp->name, dap->uda_devname) != 0) {
-			return -FI_ENODATA;
+                    !check_devname(version, dap, NULL, CHECK_DEVNAME_GETINFO)) {
+			USDF_DBG("devname of fabric not matching expected for version (version=%d) (devname='%s')\n", version, devn);
```

Comment:
Move this message into the `check_devname` function.  Log that they don't match using debug log level.